src/System.Memory/tests/ReadOnlySpan/IndexOfAny.byte.cs

```diff
@@ -36,22 +36,18 @@ public static void IndexOfAnyStrings_Byte(string raw, string search, char expect
             char[] searchFor = search.ToCharArray();
             byte[] searchForBytes = Encoding.UTF8.GetBytes(searchFor);
 
-            var index = -1;
+            var index = span.IndexOfAny(new ReadOnlySpan<byte>(searchForBytes));
```

Comment:
Nit: var => int (I realize it was like that before... but it shouldn't have been :)